Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class CaesarCipher
+    {
+        public static void Demo()
+        {
+            bool isContinue = true;
+            while (isContinue)
+            {
+                int shift = UserSelectSizeAndDirectionOfShift();
+                string forOperation = RequestInput();
+                if (shift == 99)
```

Comment:
99 is a magic number. For Caesar Cipher, we should support full extended ASCII- 255 symbols.